### ISSUE TITLE
Potential malware in the lib

### ISSUE THREAD

**Issue by codedius** [amacneil/dbmate#300]
Hi!

We're using CrowdStrike Falcon Windows Sensor (6.36.15005.) at our work and it detected suspicious indicators in the dbmate 1.15.0. Trying to install it via scoop and directly from exe - same result. Attached the report from our security department:

![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/35532915/168747188-da0a8b7b-7e6d-4c64-8356-22cb9062f803.png)

They also said that there is a high probability it might be because of files being reported to the VirusTotal site.

The hash of the file which triggered the alert: 95a70c56b297f0541b656d7249b15c22b1fe5d5422011c98e8e2b98a75baaa30

Update: have received in-depth analysis from CrowdStrike: https://www.hybrid-analysis.com/sample/95a70c56b297f0541b656d7249b15c22b1fe5d5422011c98e8e2b98a75baaa30/6285a1391064d409185d1112

**Comment by amacneil** [comment]
I am going to assume this is a false positive, unless you have more specific information about the issue?

**Comment by Enrico204** [comment]
The report says nothing strange, in my opinion. Also, there are some discrepancies between VT and Crowdstrike, as the former says that the program is contacting two Akamai IPs and the latter says that there is no network activity.

The hash is from the latest Windows version of dbmate from GitHub.

I tried to read inside both VT and Hybrid Analysis reports, no relevant issues. Perhaps these systems are too sensitive? They flagged `GetFileAttributesW` or `GetTempPathW` syscalls as "suspicious"...

However I was not able to run a test in any.run because it's limited to 32 bit software in the "zero costs" plan.

---

I want to add: I tried to build this Go "software":

```go
package main

import "fmt"

func main() {
        fmt.Println("test")
}
```

And here the results from VT: https://www.virustotal.com/gui/file/5dffe9a3234415e3a9f2430275f9a03022bde12bcad99a5d8726c328c0a38796 and Hybrid Analysis: https://www.hybrid-analysis.com/sample/5dffe9a3234415e3a9f2430275f9a03022bde12bcad99a5d8726c328c0a38796/62965fc63889e221846d2af3 . Not bad, considering that "test.go" is the most dangerous software a programmer can produce /s
